sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/implementation/LeaseStoreManagerImpl.java

```diff
@@ -314,7 +314,11 @@ public LeaseStoreManagerBuilderDefinition hostName(String hostName) {
                 this.requestOptionsFactory.createItemRequestOptions(lease),
                 serverLease ->
                 {
-                    if (serverLease.getOwner() != null && !serverLease.getOwner().equalsIgnoreCase(lease.getOwner())) {
```

Comment:
Would it be simplified it:

```
if (!lease.getOwner().equalsIgnoreCase(serverLease.getOwner())) {
```

If serverLease.getOwner() is null, the comparison simply returns false?

Reply:
yep, there's a redundant check I left when copy/paste; should be removed now